src/util/util.ts

```diff
@@ -26,6 +26,12 @@ export const HIDDEN_PLACEHOLDER = '[hidden]';
  */
 export const DEFAULT_SCROBBLE_TIME = 30;
 
+/**
+ * Percentage of track to playback before the track is scrobbled.
+ * This value is used only if scrobble percent storage is somehow corrupted.
+ */
+export const DEFAULT_SCROBBLE_PERCENT = 50;
```

Comment:
Can we use this const in `GlobalOptions` where we set to `50` to start with?

Reply:
Fixed, nice one